modtools/UnitConverter3/vsunitcsv.py

```diff
@@ -109,7 +109,7 @@ def updateUnit(self, inputRecord):
 
   def writeUnitCsv(self):
   # writes complete records again to units.csv file
-    file = open(self.unitCsvAddress, "wb")
+    file = open(self.unitCsvAddress, "w")
```

Comment:
this might need to be `wt` so it works appropriately on WIndows... Unix/Linux ignores the `t` portion.

Reply:
"t" should be the default regardless of the platform, according to the official docs, but adding it for clarity won't hurt.
Rather, by switching from the old binary mode to text, there may be issues with the encoding, whose default is platform dependent; it might be worth agreeing on one and make it explicit too with the `encoding=` parameter.

Reply:
The issue comes with Microsoft Windows and the VC Runtime as open/fopen behave differently from other platforms (see https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/fopen-wfopen?view=msvc-160); that said, we're in Python here and supposedly Pythons3 at least should do the right thing (see https://docs.python.org/3/library/functions.html#open); Py2 (https://docs.python.org/2.7/library/functions.html#open) might do the right thing too. I'm just aware of running into issues with consistency across platforms when not specifying the `t` or `b`, so I generally have leaned towards being more explicit.

Reply:
NOTE: We might want to keep it as binary so that we get consistent usage of line terminators across platforms, making the files more portable. Otherwise the underlying CLib may swap `\n` or `\r\n` (Windows) or `\n\r` (Mac) or just leave it as `\n` (Unix, Linux).

Reply:
Roger that, reverting to binary makes the most sense; I've just updated vsunitcsv.py accordingly.